clients/admin-ui/cypress/e2e/organization.cy.ts

```diff
@@ -67,7 +67,7 @@ describe("Organization page", () => {
   describe("can edit organization configuration", () => {
     beforeEach(() => {
       stubOrganizationCrud();
-      cy.visit("/management/organization");
+      cy.visit("/settings/organization");
```

Comment:
While we're in here, can we replace these hardcoded strings with references to the appropriate constants from `routes.ts` instead?

Reply:
Sure, let me change those